cms/core/tests/test_permissions.py

```diff
@@ -19,153 +131,208 @@ def create_user(cls, group_name):
         group = Group.objects.get(name=group_name)
         group.user_set.add(cls.user)
 
-    def all_permissions_check_helper(self, app: str, model: str):
-        """Helper method to check if a user has all permissions for a model."""
+        cls.user_permissions = list(cls.user.user_permissions.all() | Permission.objects.filter(group__user=cls.user))
+
+    def check_and_remove_from_user_permissions(self, app, model, permission_type):
```

Comment:
Method missing type signatures.

I'm also wondering if it should use the existing convention and be renamed to have "helper" at the end?

Reply:
[2f92347](https://github.com/ONSdigital/dis-wagtail/pull/143/commits/2f92347cabf750b124022b0c223e1b3b74ffee08)